tests/transformers/test_transformers_model_export.py

```diff
@@ -2667,155 +2606,64 @@ def test_whisper_model_serve_and_score_with_timestamps(whisper_pipeline, raw_aud
     with mlflow.start_run():
         model_info = mlflow.transformers.log_model(
             transformers_model=whisper_pipeline,
-            artifact_path=artifact_path,
-            signature=signature,
+            artifact_path="whisper_timestamps",
             model_config=model_config,
-            input_example=raw_audio_file,
+            input_example=(audio, model_config),
         )
 
-    inference_payload = json.dumps({"inputs": [base64.b64encode(raw_audio_file).decode("ascii")]})
-
-    response = pyfunc_serve_and_score_model(
-        model_info.model_uri,
-        data=inference_payload,
-        content_type=pyfunc_scoring_server.CONTENT_TYPE_JSON,
-        extra_args=["--env-manager", "local"],
-    )
-
-    values = PredictionsResponse.from_json(response.content.decode("utf-8")).get_predictions()
-    payload_output = json.loads(values.loc[0, 0])
-
-    assert (
-        payload_output["text"]
-        == mlflow.transformers.load_model(model_info.model_uri)(raw_audio_file, **model_config)[
-            "text"
-        ]
-    )
-
+    # Native transformers prediction as ground truth
+    gt = whisper_pipeline(audio, **model_config)
 
-def test_audio_classification_pipeline(audio_classification_pipeline, raw_audio_file):
-    artifact_path = "audio_classification"
-    signature = infer_signature(
-        raw_audio_file,
-        mlflow.transformers.generate_signature_output(
-            audio_classification_pipeline, raw_audio_file
-        ),
-    )
+    def _assert_prediction(pred):
+        assert pred["text"] == gt["text"]
+        for pred_chunk, gt_chunk in zip(pred["chunks"], gt["chunks"]):
```

Comment:
```suggestion
        assert len(pred["chunks"]) == len(gt["chunks"])
        for pred_chunk, gt_chunk in zip(pred["chunks"], gt["chunks"]):
```

to ensure `pred["chunks"]` and `gt["chunks"]` have the same length.

Reply:
In Python >= 3.10, the strict argument is available, but we can't use it yet:

https://docs.python.org/3/library/functions.html#zip

Reply:
great catch!